spec/factories/allocations.rb

```diff
@@ -16,5 +17,15 @@
     trait :initial do
       request_type { "initial" }
     end
+
+    trait :with_allocation_uplift do
```

Comment:
```suggestion
    trait :with_uplift do
```
Might be a little nicer for console work and when writing tests.

Reply:
Agreed, pushed changes now 👍